### ISSUE TITLE
Issue with replication group id variable

### ISSUE THREAD

**Issue by Alonreznik** [terraform-aws-modules/terraform-aws-elasticache#6]
## Description

When running the module without any `engine` parameter and `replication_group_id` parameter, i'm getting the next:

```
│ Error: Invalid combination of arguments
│ 
│   with aws_elasticache_cluster.this[0],
│   on main.tf line 49, in resource "aws_elasticache_cluster" "this":
│   49:   replication_group_id         = var.create && var.create_replication_group ? aws_elasticache_replication_group.this[0].id : var.replication_group_id
│ 
│ "replication_group_id": only one of `engine,replication_group_id` can be
│ specified, but `engine,replication_group_id` were specified.

```

It seems the engine default to `redis` somehow breaks the option of replication_group_id inside the module

**Comment by bryantbiggs** [comment]
closing due to lack of reproduction - please populate the issue template provided

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.2.2 :tada:
